backend/vaa-strapi/src/api/candidate/routes/candidate.ts

```diff
@@ -48,6 +48,7 @@ export default factories.createCoreRouter('api::candidate.candidate', {
         // Disable filters by default to avoid accidentally leaking data of relations
         restrictFilters([]),
         // Allow only updating the following fields
+        //TODO: move to the new question and answer model
         restrictBody(['gender', 'birthday', 'unaffiliated', 'photo', 'manifesto', 'motherTongues', 'otherLanguages', 'politicalExperience', 'appLanguage']),
```

Comment:
In practice, I think it should be enough to truncate this to

`restrictBody(['photo'])`

(and make `appLanguage` at the same time a prop of `User` instead of `Candidate`).